tuxemon/core/states/combat/combat.py

```diff
@@ -694,6 +710,9 @@ def end_combat(self):
         fadeout_action.parameters = [1000]
         event_engine.actions["fadeout_music"]["method"](self.game, fadeout_action)
 
+        #Round player movement in case battle started in the middle of a step
+        self.game.player1.tile_pos = (round(self.game.player1.tile_pos[0]),round(self.game.player1.tile_pos[1]))
```

Comment:
Alright if its ok with you i'll remove it from the PR. I think if that's the case we should probably wait to decide were it goes once we see what all needs to call it right?